dc-model-xml/src/main/java/de/digitalcollections/model/xml/xstream/DigitalCollectionsXStreamMarshaller.java

```diff
@@ -57,10 +64,15 @@ public DigitalCollectionsXStreamMarshaller() {
 
   private Map<String, Class> getAliases() {
     Map<String, Class> aliases = new HashMap<>();
+    aliases.put("article", ArticleImpl.class);
     aliases.put("blockquote", BlockquoteImpl.class);
     aliases.put("bulletList", BulletListImpl.class);
     aliases.put("codeBlock", CodeBlockImpl.class);
+    aliases.put("collection", CollectionImpl.class);
     aliases.put("contentBlocks", ContentBlock.class);
```

Comment:
is this right? plural for a single object?

Reply:
in v1 (below) it is singular...?

Reply:
:+1: